internal/app/singularity/cache_clean_linux.go

```diff
@@ -51,6 +66,11 @@ func cleanBlobCache() error {
 
 // CleanCache : clean a type of cache (cacheType string). will return a error if one occurs.
 func CleanCache(cacheType string) error {
+	c, err := cache.Create()
```

Comment:
Why does this need to create cache here?

Reply:
It does not really create a cache but create a cache handle. An existing cache can be used under the hood or a new one created depending on the environment variable and it it points to a new location.

Reply:
I included some comments, do you think it is enough or would prefer to switch from `Create()` to something else?

Reply:
I also just realized that I may have misunderstood your comment in the first place and slightly change the code to make it cleaner. Let me know what you think about it.

Reply:
Based on @mem, I rename the `Create()` function to `NewHandle()`, which greatly helps clarifying the situation.